RFXtrx/__init__.py

```diff
@@ -160,6 +160,19 @@ def send_stop(self, transport):
         self.cmndseqnbr = (self.cmndseqnbr + 1) % 5
         transport.send(pkt.data)
 
+    def send_program(self, transport):
+        """ Send a 'Program' command using the given transport """
+        pkt = lowlevel.Rfy()
+        pkt.set_transmit(
+            self.subtype,
+            self.cmndseqnbr,
+            self.id_combined,
+            self.unitcode,
+            0x07
+        )
+        self.cmndseqnbr = (self.cmndseqnbr + 1) % 5
+        transport.send(pkt.data)
```

Comment:
There is now too much duplication of the code in send_close, send_open, send_stop and send_program.
Could you refactor the code into a _send_cmd(self, cmd) function that all these functions can use